README.md

```diff
@@ -126,12 +128,14 @@ Flags:
 | `artifactory.scrape-uri`<br/>`ARTI_SCRAPE_URI` | No | `http://localhost:8081/artifactory` | URI on which to scrape JFrog Artifactory. |
 | `artifactory.ssl-verify`<br/>`ARTI_SSL_VERIFY` | No | `true` | Flag that enables SSL certificate verification for the scrape URI. |
 | `artifactory.timeout`<br/>`ARTI_TIMEOUT` | No | `5s` | Timeout for trying to get stats from JFrog Artifactory. |
+| `artifactory.repo-quota-enable`<br/> `ARTI_REPO_QUOTA_ENABLE` | No  | `true`  | Enables the quota monitoring. |
```

Comment:
I believe the default is `false`, not `true`

Reply:
changed from true to false